node/v2/handler.js

```diff
@@ -208,6 +220,12 @@ TChannelV2Handler.prototype.callResponseFrameHandled = function callResponseFram
     callback();
 };
 
+TChannelV2Handler.prototype.handleCancel = function handleCancel(frame, callback) {
```

Comment:
Let's at add a TODO noting that we should implement clearing of `self.streaming{Req,Res}`.

- should clear out of self.streaming{Req,Res}
- but they're currently keyed by id not tracing
- however we probably can't just shift the primary key of these collections
- rather we'll probably need to add a secondary index to protect against pathological "zero tracing" being sent... unless of course we make that a hard error

Similar concerns apply at the `TChannelConnection.requests.{in,out} layer`.